server/rest/tale.py

```diff
@@ -40,7 +41,7 @@ def __init__(self):
         self.route('DELETE', (':id',), self.deleteTale)
         self.route('GET', (':id', 'access'), self.getTaleAccess)
         self.route('PUT', (':id', 'access'), self.updateTaleAccess)
-        self.route('GET', (':id', 'export'), self.exportTale)
+        self.route('POST', (':id', 'export'), self.exportTale)
```

Comment:
Why do we need to `POST` here?

Reply:
When I was reading up on downloading zips/binaries via the browser, I saw the majority of people POSTing. Looking [here](https://stackoverflow.com/questions/16355175/which-http-method-to-use-for-file-downloading), maybe I should have used GET